packages/cli-repl/test/e2e-fle.spec.ts

```diff
@@ -111,6 +111,7 @@ describe('FLE tests', () => {
     const shell = TestShell.start({
       args: ['--nodb']
     });
+    await shell.waitForPrompt();
```

Comment:
Not sure why this worked in the first place – but this started failing because the first `shell.executeLine` thought that the initial prompt shown after startup was the prompt that came *after* the line execution, not before. This seems to pass either way with the line being added.